bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -560,20 +515,25 @@ if onsource_file:
         fap_test = sum(tot_off_snr > loud_on_bestnr)/total_trials
         pval = '< %.3g' % (1./total_trials) if fap == 0 else '%.3g' % fap
         loud_on_fap = fap
-        d = [pval, np.asarray(trig.get_end()).astype(float),\
-             trig.mass1, trig.mass2, trig.mchirp,\
-             np.degrees(trig.ra), np.degrees(trig.dec),\
-             trig.snr, trig.chisq, trig.bank_chisq,\
-             trig.cont_chisq, trig.get_null_snr()] + \
-             [trig.get_sngl_snr(ifo) for ifo in ifos] + [loud_on_bestnr]
+        d = [pval, 
+             on_trigs['network/end_time_gc'][trig_index],
+             bank_data['mass1'][on_trigs['network/template_id'][trig_index]],
+             bank_data['mass2'][on_trigs['network/template_id'][trig_index]],
+             mchirps['mchirp'][on_trigs['network/template_id'][trig_index]],
+             np.degrees(on_trigs['network/ra'][trig_index]),
+             np.degrees(on_trigs['network/dec'][trig_index]),
+             on_trigs['network/coherent_snr'][trig_index],
+             on_trigs['network/my_network_chisq'][trig_index],
+             on_trigs['network/null_snr'][trig_index]] + \
+             [on_trigs['%s/snr' % ifo][trig_index] for ifo in ifos] + [loud_on_bestnr]
         td.append(d)
     else:
         td.append(["There are no events"] + [0 for number in range(11)] + \
                   [0 for ifo in ifos] + [0])
 
     # Table header
     th = ['p-value', 'GPS time', 'Rec. m1', 'Rec. m2', 'Rec. Mc', 'Rec. RA',\
-          'Rec. Dec', 'SNR', 'Chi^2', 'Bank veto', 'Auto veto', 'Null SNR'] +\
+          'Rec. Dec', 'SNR', 'Chi^2', 'Null SNR'] +\
```

Comment:
So from here it seems that you did grab reconstructed ra and dec here.  Higher up it seemed like you hadn't.  Make sure you have them (and that you handle reconstructed spins as well).